source/community.html.haml

```diff
@@ -21,9 +21,9 @@ title: Community
             Internet Relay Chat
 
           %p
-            This is the preferred way to discuss mpv use and development. Never ask to
-            ask - just state your question. Stick around until someone replies.
-
+            This is the preferred way to discuss mpv usage and development.
```

Comment:
"use" is not incorrect and I think it sounds nicer

Reply:
restored